website/docs/docs/deploy/merge-jobs.md

```diff
@@ -13,6 +13,8 @@ By using CD in dbt Cloud, you can take advantage of deferral to build only the e
 ## Prerequisites
 - You have a dbt Cloud account. 
 - You have access to the beta release for this functionality. 
+- You have set up a [connection with your Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud). This integration lets dbt Cloud run jobs on your behalf for job triggering.
```

Comment:
double-checking: any other prereqs for Merge jobs that we should add?

Reply:
Maybe the stuff around ensuring your git provider is set to send push notifications and link to the other documentation? Although that might be overkill for here.